bake/cmd/root.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/breadtubetv/bake/providers"
```

Comment:
should this be `github.com/breadtubetv/breadtubetv/bake/providers`?

Reply:
Nah, the go formatter in VS Code automatically resolves the package refs so they're all correct.